python/lsst/ts/xml/data/sal_interfaces/MTRotator/MTRotator_Events.xml

```diff
@@ -20,6 +20,11 @@
     EnabledSubstate_Initializing=4,
     EnabledSubstate_Relative=5,
     EnabledSubstate_ConstantVelocity=6
+</Enumeration>
+  <Enumeration>
+    FaultSubstate_EmergencyStopping=0,
```

Comment:
Are you sure about having the lowest state be EmergencyStopping? this seems a bit weird to me. 

I'd suggest adding an `Unknown` state with value 0 then switch order an have `NoError=1`, `WaitClearError=2` and `EmergencyStopping=3`.

Reply:
This is to be consistent with the FaultSubStates added by Julen:
https://github.com/lsst-ts/ts_mt_hexRot_simulink/blob/develop/commonScripts/FaultSubStates.m

I think it might be better to keep the same.

Reply:
Ok, can we maybe ask them to change their state enumeration then?